.rubocop.yml

```diff
@@ -29,7 +29,7 @@ Metrics/AbcSize:
 Metrics/ClassLength:
   Max: 150
 Metrics/BlockLength:
-  IgnoredMethods: ['describe']
+  AllowedMethods: ['describe']
   Max: 30
 
 Style/Documentation:
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/207246005-de4bf07f-117b-451f-9bc1-b0a40810c00d.png)

- [ ] I am getting these errors while running rspec :fearful: Kindly fix it :blush: